scripts/cpplint.py

```diff
@@ -6365,6 +6380,27 @@ def ProcessFileData(filename, file_extension, lines, error,
 
   ResetNolintSuppressions()
 
+  # Load module dependencies
+  module_deps_file = os.path.join(os.path.dirname(filename), 'module_dependencies.txt')
+  module_deps = []
+  if os.path.isfile(module_deps_file):
+      with open(module_deps_file, 'r') as f:
+          module_deps = f.read().splitlines()
+      # strip off comments and whitespace
+      def strip_off_comments(s):
+          has_comment = s.find('#')
+          if has_comment >= 0:
+              s = s[:has_comment]
+          s = s.lstrip().rstrip()
+          return s
+      module_deps = [ strip_off_comments(module) for module in module_deps]
+      # remove empty lines
+      module_deps = [ module for module in module_deps if module]
```

Comment:
Recommend `if len(module) != 0` to make clear you're expecting strings not booleans